src/org/opensolaris/opengrok/configuration/RuntimeEnvironment.java

```diff
@@ -1561,6 +1561,9 @@ public int getPluginVersion() {
      * @return the framework
      */
     public AuthorizationFramework getAuthorizationFramework() {
```

Comment:
I think that this won't occur in our setup however yes that could cause a problem. I marked them as synchronized.